extras/kms/kms_internal_test.go

```diff
@@ -1165,7 +1165,7 @@ func TestKms_RevokeRootKeyVersion(t *testing.T) {
 				require.NoError(t, err)
 
 				testInsertEncryptedData(t, dbWrapper, rw, []byte("test-plaintext"))
-				t.Cleanup(func() { testDeleteWhere(t, db, &testEncryptedData{}, "1=1", nil) })
+				t.Cleanup(func() { testDeleteWhere(t, db, &testEncryptedData{}, "1=1") })
```

Comment:
All these little things is because of the go-dbw update, which pulls in a new (minor) version of gorm. It's impressive how many things they can break on a minor.